.circleci/config.yml

```diff
@@ -1281,73 +1278,107 @@ jobs:
-                              if ! gh pr view --json title;
-                              then
-                                echo "No PR found for this branch, stopping the job here."
-                                exit 0
-                              fi
-        
-                              # If PR state is different from OPEN
-                              if [ "$(gh pr view --json state --jq .state)" != "OPEN" ];
-                              then
-                                echo "PR is not opened, stopping the job here."
-                                exit 0
-                              fi
-                              
-                              pushd gravitee-apim-e2e/jacoco/reports
-                                export COVERAGE_SUMMARY=$(sed -nE 's/^.*<td>Total<([^>]+>){4}([^<]*)([^>]+>){4}([^<]*).*$/\2 | \4/p' index.html)
-                              popd
-        
-                              export COVERAGE_URL="https://output.circle-artifacts.com/output/job/${CIRCLE_WORKFLOW_JOB_ID}/artifacts/0/gravitee-apim-e2e/jacoco/reports/index.html"
-                              
-                              export PR_BODY_COVERAGE_SECTION="
-                              <!-- E2E Coverage placeholder -->
-                              ---
-                              ### 🧪 End-to-End Coverage
-                              
-                              | INSTRUCTIONS | BRANCHES |
-                              | :----------: | :------: |
-                              |   ${COVERAGE_SUMMARY}   |
-                              
-                              A more detailed report has been uploaded to [circleci](${COVERAGE_URL})
-                              <!-- E2E Coverage placeholder end -->
-                              "
-        
-                              export CLEAN_BODY=$(gh pr view --json body --jq .body | sed '/E2E Coverage placeholder -->/,/E2E Coverage placeholder end -->/d')
-        
-                              gh pr edit --body "$CLEAN_BODY$PR_BODY_COVERAGE_SECTION"
+                condition:
+                  and:
+                    - equal: [ v3, << parameters.execution_mode >> ]
+                    - equal: [ mongo, << parameters.database >> ]
+                steps:
+                  - persist_to_workspace:
+                      root: .
+                      paths:
+                        - gravitee-apim-gateway
+                        - gravitee-apim-rest-api
+                        - rest-api-docker-context
+                        - gateway-docker-context
+                        - gravitee-api-management
+                        - gravitee-apim-e2e/package.json
+                        - gravitee-apim-e2e/jacoco/lib
+                        - gravitee-apim-e2e/jacoco/*.xml
             - notify-on-failure
             - store_test_results:
                 path: ~/test-results
-
+    e2e-report:
+      machine:
+        image: ubuntu-2004:202101-01
```

Comment:
we could spare some credits with a docker executor and a small resource class. unfortunately, i was not able to find a docker image with ant and node in circle convenience images

Reply:
@gaetanmaisse do you have an idea ?

Reply:
We could either: 
 - install ant on a `cimg` image
 - create our own image based on `cimg` ones, I discussed that with @sebdevaux a few weeks ago

For the time being I would say we should go with what we have here